packages/block-editor/src/utils/dom.js

```diff
@@ -11,6 +11,15 @@ export function getBlockDOMNode( clientId ) {
 	return document.querySelector( '[data-block="' + clientId + '"]' );
 }
 
+export function getClockPreviewContainerDOMNode( clientId ) {
+	const domNode = getBlockDOMNode( clientId );
+	if ( ! domNode ) {
+		return;
+	}
+
+	return domNode.querySelector( '.wp-block-preview-container' );
```

Comment:
I don't think we can rely on a a new `wp-block-preview-container` className added to all blocks.
Why not just `domNode.firstChild` (I'm pretty sure it would work consistently but we could fallback to `domNode` if it's not defined.

This ties the implementation of this function to the `BlockListBlock` component but we could prevent regressions by e2e test maybe.

Reply:
I will try your suggestion of `firstChild` to fully stress test this route.

Reply:
I've implemented `firstChild`